src/site/constants/drupals.js

```diff
@@ -1,14 +1,25 @@
 const ENVIRONMENTS = require('./environments');
 
+const DRUPAL_STAGING = 'http://staging.va.agile6.com';
+
 /**
  * The decoupled-Drupal API locations by environment
  * These should be considered very temporary, because their locations will definitely change as
  * the Drupal integration continues.
  * @module site/constants/drupals
  */
-module.exports = {
-  [ENVIRONMENTS.LOCALHOST]: 'http://dev.va.agile6.com',
-  [ENVIRONMENTS.VAGOVDEV]: 'http://dev.va.agile6.com',
-  [ENVIRONMENTS.VAGOVSTAGING]: 'http://staging.va.agile6.com',
+const DRUPALS = {
+  [ENVIRONMENTS.LOCALHOST]: DRUPAL_STAGING,
+  [ENVIRONMENTS.VAGOVDEV]: DRUPAL_STAGING,
+  [ENVIRONMENTS.VAGOVSTAGING]: DRUPAL_STAGING,
   [ENVIRONMENTS.VAGOVPROD]: 'http://vagovcms.lndo.site',
 };
+
+DRUPALS.CREDENTIALS = {
+  [DRUPAL_STAGING]: {
+    username: 'api',
+    password: 'drupal8',
+  },
```

Comment:
These are the credentials we were told to use, but they don't work just yet. I confirmed with other credentials that this code does work otherwise though.